planemo/galaxy/serve.py

```diff
@@ -35,6 +37,7 @@ def _serve(ctx, runnables, **kwds):
     if port is None:
         port = network_util.get_free_port()
         kwds["port"] = port
+    kwds['galaxy_skip_client_build'] = False
```

Comment:
This should be done in cmd_serve and cmd_shed_serve I think, that way when serve is done here for workflow testing for instance this isn't rebuilt. Does that sound fair?

Reply:
Good point, fixed in a rebase!